packages/emotion/src/emotion.tsx

```diff
@@ -0,0 +1,28 @@
+import type { R2WCBaseProps } from "@r2wc/core"
+
+import createCache from "@emotion/cache"
+import { CacheProvider } from "@emotion/react"
+import { useMemo } from "react"
+
+export function withCacheProvider<Props>(
+  Component: React.ComponentType<Props>,
+  key: string = "rtwc",
```

Comment:
Is there a reason this can't be `r2wc`?

Reply:
Ah, i see: https://emotion.sh/docs/@emotion/cache#key

we need to find a way to make sure people only pass `string (Pattern: [^a-z-])`